pkg/client/k8s_client.go

```diff
@@ -0,0 +1,111 @@
+package client
+
+import (
+	"context"
+	"log"
+	"os"
+	"path/filepath"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/opensergo/opensergo-control-plane/constant"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"k8s.io/client-go/kubernetes"
+
+	"k8s.io/client-go/dynamic"
+
+	appv1 "k8s.io/api/apps/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var dynamicClient dynamic.Interface
+var client *kubernetes.Clientset
+
+func Init() error {
+	home, err := homeDir()
+	if err != nil {
+		log.Printf("Find empty home directory, %v", err)
+		return err
+	}
+	kubeConfig := filepath.Join(home, ".kube", "config")
+
+	// uses the current context get restConfig
+	config, err := clientcmd.BuildConfigFromFlags("", kubeConfig)
+	if err != nil {
+		log.Panic(err)
+	}
+	dynamicClient, err = dynamic.NewForConfig(config)
+	if err != nil {
+		log.Printf("Create dynamicClient for kubernetes failed %v", err)
+		return err
+	}
+	client, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		log.Printf("Create client set for kubernetes failed %v", err)
+		return err
+	}
+
+	return nil
+}
```

Comment:
what does this func used for? I guess it is used to check the kubeconfig when start in main.go?  If that, the k8s_operator.go has done it like following, and will return error if could not find a kubeconfig when NewKubernetesOperator() in main.go. So i think this func is unnecessary.
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/43714056/219529799-7e12eb38-6191-4c5f-bba3-b1826c05dc83.png">

Reply:
OK, I will fix it

Reply:
em... Maybe need to fix  👇 the following review comment, because I saw that the func Init() also Instance  a dynamicClient which will used in `ActiveCRDWatcher`.

Reply:
If k8sclient in ActiveCRDWatcher can be replaced, the k8s_client.go can be deleted. If not, the instance func should be reserved.